README.md

```diff
@@ -120,20 +122,113 @@ You can use Safari or Google Chrome as your default browser. If you run into a b
 [Follow the Node.js package manager install instructions](https://nodejs.org/en/download/package-manager/) for your distribution to ensure you are getting the latest version of Node.js. Many distributions only maintain an older version in their default repositories. Python 3 is only required for running machine learning (TensorFlow).
 
 ```sh
-curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
-sudo apt-get install -y nodejs npm git python3
+curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash - && sudo apt-get \
+install -y \
```

Comment:
no real reason to attempt to turn this into a one-liner. especially if the command is broken up onto multiple lines, the copy-and paste is the exact same result.

Reply:
also, in terms of clarity and readability I do prefer these all to not be jammed into one-liners

Reply:
Debian specifically, has an issue with multiline copy paste. Therefore I use oneliners to stream line the process.